### PR TITLE
Gwen writing

### PR DESCRIPTION
**Pull request by gwenchee**
This addresses https://github.com/jbae11/depletion_rom/issues/3

### CODE REVIEW

File: docs/introduction.tex [hunk 1]
```diff
@@ -3,33 +3,115 @@ \subsection{Background and Motivation}
...
+conditions such as burnup, enrichment, etc. 
+The transmutation results impact the accuracy of the \gls{UNF} 
+composition, and thus the capability of using the \gls{NFC} to 
+analyze the impact of a \gls{NFC} on variables such as waste profile.  
+
+Current fuel cycle simulator codes include \Cyclus, ORION, DYMOND 
+and VISION. 
+Each of these NFC simulators obtain transmutation results using 
+different methods. 
+Some of these codes have multiple methods for obtaining
```
Comment (jbae11) [inline docs/introduction.tex]:
```suggestion
Some of these codes have multiple options for obtaining 
```

File: docs/introduction.tex [hunk 2]
```diff
@@ -3,33 +3,115 @@ \subsection{Background and Motivation}
...
+Current fuel cycle simulator codes include \Cyclus, ORION, DYMOND 
+and VISION. 
+Each of these NFC simulators obtain transmutation results using 
+different methods. 
+Some of these codes have multiple methods for obtaining 
+transmutation results. 
+There is essentially two overall types of methods to obtain transmutation 
+results: recipe-based method and library-based method. 
+The recipe-based method is a low fidelity method, while the library-based method is 
+a high fidelity method.
```
Comment (jbae11) [inline docs/introduction.tex]: I think this sort of binary division into low and high fidelity is not the best option. Maybe just get rid of the sentence?

File: docs/introduction.tex [hunk 3]
```diff
@@ -3,33 +3,115 @@ \subsection{Background and Motivation}
...
+The recipe-based method is a low fidelity method, while the library-based method is 
+a high fidelity method. 
+In the recipe-based method, direct neutronics calculations are not performed 
+within the model, it is done externally \cite{yacout_vision_2006}. 
+The resulting fresh and spent fuel compositions for specific parameters 
+is entered directly into the \gls{NFC} model \cite{sunny_transition_2015}. 
+In the library-based method,  the \gls{NFC} code dynamically calculates depleted 
+fuel recipes by interpolating reactor data libraries generated by
+reactor physics burnup calculation code such as SERPENT 
+\cite{leppanen_serpent_2013}, Origen \cite{croff_users_1980}, etc.
```
Comment (jbae11) [inline docs/introduction.tex]:
```suggestion
\cite{leppanen_serpent_2013}, \cite{ORIGEN} \cite{croff_users_1980}, and etc. 
```

File: docs/introduction.tex [hunk 4]
```diff
@@ -3,33 +3,115 @@ \subsection{Background and Motivation}
...
+        \Cyclus & Recipe-based and Library-based\\
+        ORION & Recipe-based and Library-based\\
+        DYMOND & Recipe-based  \\
+        VISION & Recipe-based  \\
+        \hline
+    \end{tabular}
+    \caption{\gls{NFC} Methods for transmutation in reactor module}
+\end{table}
+
+\Cyclus has three methods to obtain transmutation results: \Cycamore
```
Comment (jbae11) [inline docs/introduction.tex]:
```suggestion
\Cyclus has three methods to obtain transmutation results: the \Cycamore 
```

File: docs/introduction.tex [hunk 5]
```diff
@@ -3,33 +3,115 @@ \subsection{Background and Motivation}
...
+        ORION & Recipe-based and Library-based\\
+        DYMOND & Recipe-based  \\
+        VISION & Recipe-based  \\
+        \hline
+    \end{tabular}
+    \caption{\gls{NFC} Methods for transmutation in reactor module}
+\end{table}
+
+\Cyclus has three methods to obtain transmutation results: \Cycamore 
+recipe reactor \cite{huff_extensions_2014} CyBORG
```
Comment (jbae11) [inline docs/introduction.tex]:
```suggestion
recipe reactor \cite{huff_extensions_2014}, CyBORG 
```

File: docs/introduction.tex [hunk 6]
```diff
@@ -3,33 +3,115 @@ \subsection{Background and Motivation}
...
+        DYMOND & Recipe-based  \\
+        VISION & Recipe-based  \\
+        \hline
+    \end{tabular}
+    \caption{\gls{NFC} Methods for transmutation in reactor module}
+\end{table}
+
+\Cyclus has three methods to obtain transmutation results: \Cycamore 
+recipe reactor \cite{huff_extensions_2014} CyBORG 
+\cite{skutnik_cyborg_2016} and Bright-lite \cite{flanagan_brightlite_2014}.
```
Comment (jbae11) [inline docs/introduction.tex]:
```suggestion
\cite{skutnik_cyborg_2016}, and Bright-lite \cite{flanagan_brightlite_2014}.  
```

File: docs/introduction.tex [hunk 7]
```diff
@@ -3,33 +3,115 @@ \subsection{Background and Motivation}
...
+    \end{tabular}
+    \caption{\gls{NFC} Methods for transmutation in reactor module}
+\end{table}
+
+\Cyclus has three methods to obtain transmutation results: \Cycamore 
+recipe reactor \cite{huff_extensions_2014} CyBORG 
+\cite{skutnik_cyborg_2016} and Bright-lite \cite{flanagan_brightlite_2014}.  
+The \Cycamore recipe reactor accepts a fresh and spent fuel recipe that is 
+defined by the user. 
+CyBORG calls the Origen solver \gls{API} directly to calculate depleted fuel
```
Comment (jbae11) [inline docs/introduction.tex]:
```suggestion
CyBORG calls the \cite{ORIGEN} solver \gls{API} directly to calculate depleted fuel
```

File: docs/introduction.tex [hunk 8]
```diff
@@ -3,33 +3,115 @@ \subsection{Background and Motivation}
...
+compositions, which then becomes the output recipe for the CyBORG reactor
```
Comment (jbae11) [inline docs/introduction.tex]:
```suggestion
compositions, which then becomes the output fuel composition.
```

File: docs/introduction.tex [hunk 9]
```diff
@@ -3,33 +3,115 @@ \subsection{Background and Motivation}
...
+\end{table}
+
+\Cyclus has three methods to obtain transmutation results: \Cycamore 
+recipe reactor \cite{huff_extensions_2014} CyBORG 
+\cite{skutnik_cyborg_2016} and Bright-lite \cite{flanagan_brightlite_2014}.  
+The \Cycamore recipe reactor accepts a fresh and spent fuel recipe that is 
+defined by the user. 
+CyBORG calls the Origen solver \gls{API} directly to calculate depleted fuel
+compositions, which then becomes the output recipe for the CyBORG reactor 
+in \Cyclus \cite{skutnik_cyborg_2016}.
```
Comment (jbae11) [inline docs/introduction.tex]:
```suggestion
```

File: docs/introduction.tex [hunk 10]
```diff
@@ -3,33 +3,115 @@ \subsection{Background and Motivation}
...
+
+\Cyclus has three methods to obtain transmutation results: \Cycamore 
+recipe reactor \cite{huff_extensions_2014} CyBORG 
+\cite{skutnik_cyborg_2016} and Bright-lite \cite{flanagan_brightlite_2014}.  
+The \Cycamore recipe reactor accepts a fresh and spent fuel recipe that is 
+defined by the user. 
+CyBORG calls the Origen solver \gls{API} directly to calculate depleted fuel
+compositions, which then becomes the output recipe for the CyBORG reactor 
+in \Cyclus \cite{skutnik_cyborg_2016}. 
+Bright-lite uses libraries that represent a specific reactor design to
```
Comment (jbae11) [inline docs/introduction.tex]: this is quite unclear. How does it calculate the burnup? Maybe it helps to illustrate what the user puts in and how the algorithm uses the user inputs, and what the algorithm outputs

File: docs/introduction.tex [hunk 11]
```diff
@@ -3,33 +3,115 @@ \subsection{Background and Motivation}
...
+in \Cyclus \cite{skutnik_cyborg_2016}. 
+Bright-lite uses libraries that represent a specific reactor design to 
+calculate the burnup, and output isotopic vectors for a given input fuel
+\cite{flanagan_brightlite_2014}. 
+ORION has a recipe-based and library-based method to obtain transmutation results. 
+In ORION's library-based method, burnup-dependent cross section libraries 
+for multiple reactor types with multiple initial fuel enrichment are 
+generated before ORION analysis \cite{sunny_transition_2015}. 
+The libraries are then used to generate spent fuel recipes based on 
+reactor conditions.
```
Comment (jbae11) [inline docs/introduction.tex]: what conditions? like power?

File: docs/introduction.tex [hunk 12]
```diff
@@ -3,33 +3,115 @@ \subsection{Background and Motivation}
...
+Bright-lite uses libraries that represent a specific reactor design to 
+calculate the burnup, and output isotopic vectors for a given input fuel
+\cite{flanagan_brightlite_2014}. 
+ORION has a recipe-based and library-based method to obtain transmutation results. 
+In ORION's library-based method, burnup-dependent cross section libraries 
+for multiple reactor types with multiple initial fuel enrichment are 
+generated before ORION analysis \cite{sunny_transition_2015}. 
+The libraries are then used to generate spent fuel recipes based on 
+reactor conditions. 
+DYMOND has a recipe-based method where fuel cycle recipes for both the input
```
Comment (jbae11) [inline docs/introduction.tex]:
```suggestion
DYMOND has a recipe-based method where recipes for both the input 
```

File: docs/introduction.tex [hunk 13]
```diff
@@ -3,33 +3,115 @@ \subsection{Background and Motivation}
...
+for multiple reactor types with multiple initial fuel enrichment are 
+generated before ORION analysis \cite{sunny_transition_2015}. 
+The libraries are then used to generate spent fuel recipes based on 
+reactor conditions. 
+DYMOND has a recipe-based method where fuel cycle recipes for both the input 
+and output fuel compositions are externally calculated. 
+Isotopes are tracked in lumped categories: fission products, minor 
+actinides, Uranium and Plutonium \cite{feng_standardized_2016}.  
+VISION is an extention of the DYMOND code. 
+It has a similar recipe-based method as the DYMOND code, but with isotopic
```
Comment (jbae11) [inline docs/introduction.tex]:
```suggestion
It has a similar recipe-based method as the DYMOND code, but with individual isotopic 
```

File: docs/introduction.tex [hunk 14]
```diff
@@ -3,33 +3,115 @@ \subsection{Background and Motivation}
...
+Isotopes are tracked in lumped categories: fission products, minor 
+actinides, Uranium and Plutonium \cite{feng_standardized_2016}.  
+VISION is an extention of the DYMOND code. 
+It has a similar recipe-based method as the DYMOND code, but with isotopic 
+tracking \cite{yacout_vision_2006}. 
+
+For \gls{NFC} simulators, striking a balance between fidelity 
+and computational cost is a key issue. 
+The advantage of using high fidelity models that have a physics-based 
+approach is its inherent flexibility to readily accommodate a broad
```
Comment (jbae11) [inline docs/introduction.tex]: This paragraph is weird.. isn't it? Maybe cut it into many sentences?

File: docs/introduction.tex [hunk 15]
```diff
@@ -3,33 +3,115 @@ \subsection{Background and Motivation}
...
+The advantage of using low fidelity models (recipe-based method)
+is the low computational cost. 
+It is an acceptable method for modeling fuel cycles with fixed input 
+composition or fuel cycles at equilibrium \cite{sunny_transition_2015}. 
+However, for fuel cycles not at equilibrium, it can result in less 
+accurate results. 
+
+Therefore, to find a middle ground of accurate depletion data while 
+maintaining a practical computational cost, this paper introduces 
+a reduced order model that is able to predict \gls{PWR} \gls{UNF}
```
Comment (jbae11) [inline docs/introduction.tex]:
```suggestion
a trained neural network model that is able to predict \gls{PWR} \gls{UNF}
```

**Comment by gwenchee** [comment]
i've made the edits for all the comments except the bright-lite one. We can merge it for now, and i'll ponder about how to write a better description for bright-lite.

**Pull request merged**